Makefile

```diff
@@ -13,12 +13,28 @@
 
 all: release
 
+# Improve build speeds during development by removing the --no-cache flag
+DOCKER_BUILD_FLAGS=--no-cache
+
 .PHONY: release
-release: linux-plugins
+release: build build-init
+	docker system prune -f
+	docker build $(DOCKER_BUILD_FLAGS) -t amazon/aws-for-fluent-bit:main-release -f ./scripts/dockerfiles/Dockerfile.main-release .
+	docker tag amazon/aws-for-fluent-bit:main-release amazon/aws-for-fluent-bit:latest
```

Comment:
why the extra tagging step? Why not just directly build it and tag it as `latest`??

Reply:
That's probably a good idea. I can change it.

Reply:
Actually, it's because the Dockerfile.init-release depends on main-release. That's why we build main-release, and tag with latest